docs/programming_guide/provisioning_system.rst

```diff
@@ -397,13 +397,21 @@ Edit the project.yml configuration file to meet your project requirements:
 Default project.yml file
 ========================
 
-The following is an example of the default project.yml file.
+The following is an example of the default project.yml file of HA mode.
 
-.. literalinclude:: ../../nvflare/lighter/project.yml
+.. literalinclude:: ../../nvflare/lighter/hd_project.yml
```

Comment:
Is hd_project.yml supposed to be ha_project.yml?

Reply:
Good catch.  Thanks.  Let me fix it.